### PR TITLE
feat(init): Add initial files to bootstrap project

### PR DESCRIPTION
**Pull request by tainhenning**

### CODE REVIEW

**Comment by jdhouti** [comment]
Add me as contributor so that I can approve

File: fyi_links.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "salary_link": "https://www.levels.fyi/company/Netflix/salaries/Software-Engineer/",
```
Comment (jdhouti) [inline fyi_links.json:2]: Is this for testing or should there be a placeholder for the company name in the URL?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from selenium import webdriver
+import json
+import constants
+
+def main():
+    with open(constants.JSON_PATH) as f:
+        fyi_links = json.load(f)
+    driver = webdriver.Chrome()
+    driver.get(fyi_links[constants.NETFLIX_SALARY_LINK])
```
Comment (jdhouti) [inline main.py:9]: Can we abstract this?

**Review by jdhouti — APPROVED** [review]
Looks good. Better see some tests. I'll work on Jenkins pipeline tomorrow

File: fyi_links.json (hunk 1 above)
Comment (tainhenning) [reply fyi_links.json:2]: It is for testing, we should probably align on a JSON format though that would be hosted on S3 before moving forward, any suggested formats would be greatly appreciated.

File: main.py (hunk 2 above)
Comment (tainhenning) [reply main.py:9]: Depends... it is good form to keep constants out of logic files... in java, unsure if that is still a best practice in python. Would still want to keep this data in JSON format though in prep for scaling.

**Comment by Mohit-Doshi** [comment]
def LGTM 🚢 🚢 🚢 🚢

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Python
+.env
+__pycache__
```
Comment (Mohit-Doshi) [inline .gitignore:3]: Let's also add a `.DS_Store` file 🍎

File: README.md [hunk 4]
```diff
@@ -1,2 +1,9 @@
 # levels-slack-scrape
 TC updates whenever you want
+
+# how to use
+1. Make sure you have chromedriver in your PATH (depends on OS)
```
Comment (Mohit-Doshi) [inline README.md:5]: 464 gang gang

**Pull request merged**